EXAMPLE.md

```diff
@@ -413,17 +414,14 @@ from pydantic import Field
 
 from .base_model import BaseModel
 from .enums import Color
+from .fragments import BasicUser, UserPersonalData
```

Comment:
It's name of the [fragment](https://github.com/mirumee/ariadne-codegen/pull/143/files/ebdd09f5a544872704e06df81dd7d8fb7b49e82c#diff-74342ba78c423722986ffbe5511c7aff26bec42599cef83349a4be7e5506f237R106)

These classes are generated in the same way as the classes for the result of an operation, but instead of the name of the operation, the name of the fragment is used. In addition, they are all placed in the same file, rather than separated by individual fragments/operations.